src/app/parcels/ActionBar/ActionModals/GenerateMapModal.tsx

```diff
@@ -25,18 +25,16 @@ const GenerateMapModal: React.FC<ActionModalProps> = (props) => {
         setActionCompleted(true);
     };
 
-    const formattedPostcodes = props.selectedParcels.reduce<string[]>(
-        (formattedPostcodes, parcel) => {
+    const formattedPostcodes = Array.from(
+        props.selectedParcels.reduce<Set<string>>((formattedPostcodes, parcel) => {
             if (parcel.addressPostcode) {
```

Comment:
Is there any way of making this more readable?

Reply:
I've moved the uniqueness finding into separate statement to make it clearer what's going on. The formatting code is now unchanged! Lmk if there are any other improvements you can think of :)

Reply:
Looks great! Let's wait for someone else to review before merging 👍